app/helpers/application_helper.rb

```diff
@@ -320,4 +320,8 @@ def get_read_access_token(study, user)
 			user.valid_access_token[:access_token]
 		end
 	end
+
+	def pluralize_without_count(count, noun, text = nil)
```

Comment:
No spaces around `=` in default arguments:
```suggestion
	def pluralize_without_count(count, noun, text=nil)
```